src/machine-config.h

```diff
@@ -167,29 +167,26 @@ using virtio_device_config = std::variant<virtio_console_config, ///< Console
 /// \brief List of VirtIO devices
 using virtio_configs = boost::container::static_vector<virtio_device_config, VIRTIO_DEVICE_MAX>;
 
-/// \brief Rollup configuration
-struct rollup_config {                                       // NOLINT(bugprone-exception-escape)
-    memory_range_config rx_buffer{0x60000000, 2 << 20};      ///< RX buffer
-    memory_range_config tx_buffer{0x60200000, 2 << 20};      ///< TX buffer
-    memory_range_config input_metadata{0x60400000, 4096};    ///< Buffer for input metadata
-    memory_range_config voucher_hashes{0x60600000, 2 << 20}; ///< Buffer for the voucher hash array
-    memory_range_config notice_hashes{0x60800000, 2 << 20};  ///< Buffer for the notice hash array
+/// \brief Cmio configuration
+struct cmio_config {                                    // NOLINT(bugprone-exception-escape)
+    memory_range_config rx_buffer{0x60000000, 2 << 20}; ///< RX buffer
```

Comment:
We should use a constants here, instead of hard-coded value.
Same for the buffer size, so it will be easier to change this in the future.

Actually I see 0x60000000 and 0x60200000 in a some test Lua scripts, maybe we could expose a Lua variable with these value it.

Reply:
Perna has a PR where these numbers are not configurable anymore and are exposed to Lua. I think it is still WIP and I am sure he has a more recent push in his computer. But this should be it. https://github.com/cartesi/machine-emulator/pull/192